pkg/services/applicationv2/remover_test.go

```diff
@@ -0,0 +1,44 @@
+package applicationv2
+
+import (
+	"context"
+
+	"github.com/weaveworks/weave-gitops/pkg/gitproviders/gitprovidersfakes"
+
+	"github.com/weaveworks/weave-gitops/pkg/git"
+	"github.com/weaveworks/weave-gitops/pkg/git/wrapper/wrapperfakes"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Remover", func() {
+	Describe(".Remove()", func() {
+		It("removes application auto-merge true", func() {
+			ctx := context.Background()
+
+			fakeGit := &wrapperfakes.FakeGit{}
+			fakeGitClient := git.New(nil, fakeGit)
+
+			orgProvider := &gitprovidersfakes.FakeGitProvider{}
+
+			remover := NewRemover(fakeGitClient, orgProvider)
+
+			err := remover.Remove(ctx, "my-app", "wego-system", true)
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
We need to check on what other side effects we think should or shouldn't happen. Since this is `auto-merge` can we check the local git repository to make sure it equals our expected state?